packages/angular/src/generators/library/library.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  formatFiles,
+  installPackagesTask,
+  moveFilesToNewDirectory,
+  Tree,
+} from '@nrwl/devkit';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+import { jestProjectGenerator } from '@nrwl/jest';
+import { Linter } from '@nrwl/workspace';
+
+import init from '../../generators/init/init';
+import addLintingGenerator from '../add-linting/add-linting';
+import karmaProjectGenerator from '../karma-project/karma-project';
+
+import { addModule } from './lib/add-module';
+import { normalizeOptions } from './lib/normalize-options';
+import { updateLibPackageNpmScope } from './lib/update-lib-package-npm-scope';
+import { updateProject } from './lib/update-project';
+import { updateTsConfig } from './lib/update-tsconfig';
+import {
+  enableStrictTypeChecking,
+  setLibraryStrictDefault,
+} from './lib/enable-strict-type-checking';
+import { NormalizedSchema } from './lib/normalized-schema';
+import { Schema } from './schema';
+import { UnitTestRunner } from '../../utils/test-runners';
+
+export async function libraryGenerator(host: Tree, schema: Partial<Schema>) {
+  // Create a schema with populated default values
+  const mergedSchema: Schema = {
+    buildable: false,
+    enableIvy: false,
+    linter: Linter.EsLint,
+    name: '', // JSON validation will ensure this is set
+    publishable: false,
+    simpleModuleName: false,
+    skipFormat: false,
+    unitTestRunner: UnitTestRunner.Jest,
+    ...schema,
+  };
+  // Do some validation checks
+  const options = normalizeOptions(host, mergedSchema);
+  if (!options.routing && options.lazy) {
+    throw new Error(`routing must be set`);
+  }
+
+  if (options.enableIvy === true && !options.buildable) {
+    throw new Error('enableIvy must only be used with buildable.');
+  }
+
+  if (options.publishable === true && !mergedSchema.importPath) {
+    throw new Error(
+      `For publishable libs you have to provide a proper "--importPath" which needs to be a valid npm package name (e.g. my-awesome-lib or @myorg/my-lib)`
+    );
+  }
+
+  await init(host, {
+    ...options,
+    skipFormat: true,
+  });
+
+  const runAngularLibrarySchematic = wrapAngularDevkitSchematic(
+    '@schematics/angular',
+    'library'
+  );
+  await runAngularLibrarySchematic(host, {
+    name: options.name,
+    prefix: options.prefix,
+    entryFile: 'index',
+    skipPackageJson: !(options.publishable || options.buildable),
+    skipTsConfig: false,
```

Comment:
I believe I made this change because the tsconfig.json files were not being created but we then needed them.

I'll check if setting it back to true causes any issues.